templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,15 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 41**
+
+* [`GET users/me/subscriptions`](/api/get-subscriptions): Added `stream_admins`
+  field to Subscription objects representing the list of user ids of stream
+  admins of the stream.
+
+* [`GET /events`](/api/get-events): Added `stream_admins` field to Subscription
+  objects sent in `subscriptions` events.
```

Comment:
Let's move this commit earlier in the branch, before any code that makes it possible to create stream administrators (including when one creates a new stream); that's generally the correct merge order.